src/components/DataLabel.tsx

```diff
@@ -1,13 +1,15 @@
 import React from 'react';
 
-export type DataLabelProps = {
+import { PropsWithElementAttributes } from '../utils';
+
+export type DataLabelProps = PropsWithElementAttributes<{
   data?: string | number;
   label?: string;
-};
+}>;
 
-export const DataLabel = ({ data = 'data', label = 'label' }: DataLabelProps) => {
+export const DataLabel = ({ data = 'data', label = 'label', className = 'raf-data-label', style }: DataLabelProps) => {
   return (
-    <div className="raf-data-label">
+    <div className={className} style={style}>
       <span className="raf-data-label__label">{label}</span>
       <span className="raf-data-label__data">{data}</span>
```

Comment:
should we always keep the default classnames instead of replacing them by given prop? lots of sub-elements reuse the parent classname like this

Reply:
That's a good point actually, I haven't thought this one through. I don't really see any drawbacks to keeping the original one - you'd still be able to override the styling and final render would make sense in terms of class name hierarchy. I was following (kind of) the [solution](https://github.com/GetStream/stream-chat-react/blob/76659fce2c0212bfa162229b8286f202cab87405/src/components/MessageList/MessageList.tsx#L204-L220) which was recently added to the chat where they also override the default ones.

Reply:
It makes sense to keep the default styles everywhere and let customers easily override them. Imagine you want to change background color of a component but this way you end up writing(copying into your code) 100 lines of css for it

Reply:
No I fully agree, will add these changes tomorrow. 👍